documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -563,7 +563,18 @@ Test run finished after 64 ms
 [         0 tests failed          ]
 ----
 
+You can also https://docs.oracle.com/javase/tutorial/deployment/jar/run.html[run] the
```

Comment:
Why does this link to the docs for using `-jar`?

Reply:
It should not.  I mistakenly copied it from the existing "you can run" text above (line 532).